private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -243,6 +245,13 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		"",
 		`The config file or data to use.`,
 	)
+	flagSet.StringSliceVar(
+		&f.ExcludePaths,
+		excludePathsFlagName,
+		nil,
+		`Exclude these paths from the generation process. This can be proto files or directories.
+If specified multiple times, the union will be taken.`,
+	)
```

Comment:
We probably want to take inspiration from `bufcli.BindPathsAndDeprecatedFiles` here:

```
	flagSet.StringSliceVar(
		pathsAddr,
		pathsFlagName,
		nil,
		`Limit to specific files or directories, for example "proto/a/a.proto" or "proto/a".
If specified multiple times, the union will be taken.`,
	)

```

Should we just do something like:
```suggestion
	flagSet.StringSliceVar(
		&f.ExcludePaths,
		excludePathsFlagName,
		nil,
		`Exclude specific files or directories, for example "proto/a/a.proto" or "proto/a".
If specified multiple times, the union will be taken.`,
	)
```